interfaces/api/v1/core/settings.py

```
@@ -173,6 +183,129 @@ def update_embedding_config(body: EmbeddingConfigUpdate):
+    url = f"{base_url.rstrip('/')}/chat/completions"
+    headers = {
+        "Authorization": f"Bearer {api_key}",
+        "Content-Type": "application/json"
+    }
+    # 尝试最简单的请求格式
+    payload = {
+        "model": "gpt-3.5-turbo",
+        "messages": [{"role": "user", "content": "Hi"}]
+    }
+    try:
+        async with httpx.AsyncClient(timeout=10) as client:
```

Ruff (0.15.10)

[error] Undefined name `httpx`

(F821)